benchmarks/OptimizationCUTEst/CUTEst.jmd

```diff
@@ -140,12 +142,70 @@ Next, we examine the same relationship for problems with inequality-constrained
 neq_unb_problems = CUTEst.select(min_con=1, only_ineq_con=true, only_free_var=true)
 
 # Analysis
-neq_unb_results =  run_benchmarks(unc_problems, optimizers)
+neq_unb_results =  run_benchmarks(neq_unb_problems, optimizers)
 
 @df neq_unb_results scatter(:n_vars, :secs,
         group = :solver,
         xlabel = "n. variables",
         ylabel = "secs.",
         title = "Time to solution by optimizer and number of vars",
     )
+```
+
+## Equality/Inequality constrained problems with bounded variables
+
+Now we analyze the subset of problems with equality/inequality constraints and whose
+variables are bounded. There are 666 such problems.
+
+The following figure shows the results of the same benchmarks previously described for the
+problems on this section.
+
+```julia
+eq_bou_problems = CUTEst.select(min_con=1, only_equ_con=true, only_free_var=false)
```

Comment:
`CUTEst.select` is still available for backward compatiblity but the exported function is `select_sif_problems`.
https://jso.dev/CUTEst.jl/dev/classification/#CUTEst.select_sif_problems

Reply:
Thanks for the tip. Should I open an issue in CUTEst.jl (or the [JSO](https://jso.dev/tutorials/introduction-to-cutest/) page) to update the documentation?

Reply:
Open an issue please. I updated the documentation of CUTEst.jl but I forgot the tutorial.